sequence_classification/results_presenter.py

```diff
@@ -80,3 +84,52 @@ def show_box_plots(self):
     @staticmethod
     def calc_accuracy_from_cm(cm):
         return cm.diagonal().sum() / cm.sum()
+
+    def prepare_measurements(self):
+        if self.measurements:
+            return
```

Comment:
Jeśli `prepare_measurements` ma byc wywoływane tylko raz w `__init__`, to możemy pozbyć się tego sprawdzania lub rzucić tutaj wyjątek w stylu: `raise NameError('self.results is not set')`.